libflux/flux-core/src/semantic/mod.rs

```diff
@@ -537,6 +537,10 @@ pub enum Feature {
     /// Enables calls to map to be vectorized when the function contains
     /// float casts.
     VectorizedFloat,
+
+    /// Enables calls to map to be vectorized when the function contains
+    /// unary operators like: add, sub exists, not.
+    VectorizedUnaryOps,
```

Comment:
It might be good to start generating Rust code for features at some point. (just an observation)

Reply:
Building the enum from the yaml file? Yeah, that's an interesting idea. It could be something we do via `build.rs`